CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Features ✨ and improvements 🏁
 * Optimise the bearing update frequency for the location puck animator. ([1398](https://github.com/mapbox/mapbox-maps-android/pull/1398))
 * Use `orientation` model source property to update the 3D puck's bearing, as it is more efficient than updating the `model-rotation` layer property. ([1407](https://github.com/mapbox/mapbox-maps-android/pull/1407))
+* Remove deprecated `FollowPuckViewportStateOptions.animationDurationMs` from experimental viewport plugin. ([1421](https://github.com/mapbox/mapbox-maps-android/pull/1421))
```

Comment:
guess it's still a breaking change

Reply:
we are allowed to make breaking changes for experimental APIs.

Reply:
or do you mean we have a breaking change section? I'd prefer not to have a breaking change section as it would cause confusion to the customers we are not following semver. This is an experimental feature, and we are still iterating on it.

Reply:
Lets align approach here since Kiryl put other experimental api removal changelog note to a breaking change section. 
I'd vote for the `breaking change` section, from the changelog entry customers will see that we only change experimental API and not a stable one.

Reply:
ok, we can aligned on that approach